kotest-property/src/jvmTest/kotlin/com/sksamuel/kotest/property/ForAll2Test.kt

```diff
@@ -121,4 +121,18 @@ Repeat this test by using seed 1234
 
 Caused by IllegalArgumentException: something unexpected happened"""
    }
+
+   test("forAll with 2 arbs should skip first 4 tests") {
+      shouldThrowAny {
+         forAll(
+            config = PropTestConfig(skipTo = 5, seed = 5847062201763421121),
```

Comment:
In real life usage, how do we know how many tests to skip? Do we need to document that somewhere?

Reply:
As I'm not that familiar with this case, just an idea: If Kotest currently prints
```
Property failed after 18 attempts
```
we could change that message to
```
Property failed after 18 attempts. Use `config = PropTestConfig(skipTo = 17)` to trigger the failed case directly.
```

That kind of message, tailored to the actual case, helps more than just burying a hint in the docs.

Reply:
> As I'm not that familiar with this case, just an idea: If Kotest currently prints
> 
> ```
> Property failed after 18 attempts
> ```
> 
> we could change that message to
> 
> ```
> Property failed after 18 attempts. Use `config = PropTestConfig(skipTo = 17)` to trigger the failed case directly.
> ```
> 
> That kind of message, tailored to the actual case, helps more than just burying a hint in the docs.

I like this suggestion, but IMO "Property failed after 18 attempts" means "it succeeded 17 times, then failed", and so if should be " Use `config = PropTestConfig(skipTo = 18)` to trigger the failed case directly." because generally `skipTo = 17` means "skip 16 cases, and do case 17 and onwards"

Reply:
So what's the consensus here, should we change the message?

Reply:
IMO we are good here, "skipTo = 5" does mean "skip first 4 tests"